getImportantAnimals

```diff
@@ -0,0 +1,17 @@
+# get most talked about animals
+
+def getImportantAnimals(keepAnimals_dic):
+    if len(keepAnimals_dic) <= 2:
+        Animal_ls =list(keepAnimals_dic.keys())
```

Comment:
What does `ls` stand for? Also, your variable name should start with a lowercase letter.

Since the method is getImportantAnimals, how about renaming this variable to `importantAnimals` instead?